app/views/auth/ImportAccountView/ImportAccountForm.tsx

```diff
@@ -37,7 +38,8 @@ export const importAccountFormOnSubmit = async (
   const { setStatus, setErrors, setSubmitting } = helpers;
 
   try {
-    await importAccount(accountName, entropy, password);
+    const decodedEntropy = bip39.mnemonicToEntropy(entropy);
```

Comment:
Sorry if I was confusing! We need to be backwards compatible for all users who have saved their `hexEntropy` from previous wallets.

So the import account form takes either `mnemonicEntropy` or `hexEntropy`.  If it is hex, it will import it directly. If it is mnemonic, it will convert to hex then import.

The validations will be a little trickier! Sorry about that!

You are correct that we will only show the `mnemonicEntropy` from here on out. But we want users to have the option to enter the hex.